Description: Do not install wxstd.mo files
Author: Kamal Mostafa <kamal@whence.com>
Bug-Debian: http://bugs.debian.org/813344

These files are supplied by the package wx3.0-i18n in Debian; this package
must not overwrite them.

--- trustedqsl-2.2.orig/apps/CMakeLists.txt
+++ trustedqsl-2.2/apps/CMakeLists.txt
@@ -155,15 +155,9 @@ if(NOT APPLE AND NOT WIN32)
 	endif()
 	install(FILES icons/key48.png DESTINATION share/pixmaps RENAME TrustedQSL.png)
 	install(FILES lang/de/tqslapp.mo DESTINATION share/locale/de/LC_MESSAGES)
-	install(FILES lang/de/wxstd.mo DESTINATION share/locale/de/LC_MESSAGES)
 	install(FILES lang/es/tqslapp.mo DESTINATION share/locale/es/LC_MESSAGES)
-	install(FILES lang/es/wxstd.mo DESTINATION share/locale/es/LC_MESSAGES)
 	install(FILES lang/fr/tqslapp.mo DESTINATION share/locale/fr/LC_MESSAGES)
-	install(FILES lang/fr/wxstd.mo DESTINATION share/locale/fr/LC_MESSAGES)
 	install(FILES lang/it/tqslapp.mo DESTINATION share/locale/it/LC_MESSAGES)
-	install(FILES lang/it/wxstd.mo DESTINATION share/locale/it/LC_MESSAGES)
 	install(FILES lang/ja/tqslapp.mo DESTINATION share/locale/ja/LC_MESSAGES)
-	install(FILES lang/ja/wxstd.mo DESTINATION share/locale/ja/LC_MESSAGES)
 	install(FILES lang/pt/tqslapp.mo DESTINATION share/locale/pt/LC_MESSAGES)
-	install(FILES lang/pt/wxstd.mo DESTINATION share/locale/pt/LC_MESSAGES)
 endif()
